client/src/pages/Login.js

```diff
@@ -44,6 +43,14 @@ export default function Login({ state, dispatch }) {
     }
   };
 
+  const guestLogin = () => {
+    const guestEmail = "guest@guest.guest";
+    const guestPassword = "guest2020";
+    userLogin(guestEmail, guestPassword, dispatch).catch((error) => {
+      setLoginError(error.message);
+    });
+  };
```

Comment:
this is good! we could also just fill up the textfields when guest is clicked and using the login button to log guest into the app. setLoginEmail and setLoginPassword to those variables and set values for the textfields to be loginEmail and loginPassword. I think letting guest use the login button would be better for a demo.